File: src/main/Java/boj_step8/boj2869.java
import java.io.*;
import java.util.StringTokenizer;

public class boj2869 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int a = Integer.parseInt(tokenizer.nextToken());
        int b = Integer.parseInt(tokenizer.nextToken());
        int v = Integer.parseInt(tokenizer.nextToken());
        int days = ((v - a) % (a - b) != 0) ? (v - a) / (a - b) + 2 : (v - a) / (a - b) + 1;
        writer.append(String.valueOf(days));
        writer.close();
    }
}

File: src/main/kotlin/boj2869.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.net.DatagramSocket
import java.time.temporal.ValueRange

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val (a, b, v) = r.readLine().split(" ").map { it.toInt() }
    var tmp = 0
    var days = if((v - a) % (a -b) != 0) (v - a) / (a - b) + 2 else (v - a) / (a - b) + 1
    w.write(days.toString())
    w.close()
}